Worktez/meet.worktez
# feat(Auth): Added Authentication

Functionality:
Added Authentication for Meets
Added Email and Name for Jitsi APIs

Solution:
Added Authentication for Meets
Sent Email and Name for Jitsi APIs

Risk level:

How to test:
Try Logging in into Meet.worktez
Once you join a meet check whether the name and email is autofilled.
